controllers/shooter.py

```diff
@@ -27,65 +27,85 @@ class ShooterController(StateMachine):
 
     def __init__(self) -> None:
         # set default preference for goal height
-        pass
+        self.GoalHeight_Preference = GoalHeight.HIGH
```

Comment:
All variable names must be snake case, per [our style guide](https://wiki.thedropbears.org.au/sections/software/coding-standards). They must _not_ be a mixture of camel case and snake case.